CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.16.1] - 2018-11-27
+### Added
```

Comment:
Internally this is added -- externally this is a bug that's been fixed. Change to ```### Fixed`